hi-ml/src/health_ml/experiment_config.py

```diff
@@ -10,6 +10,7 @@ class ExperimentConfig(param.Parameterized):
     model: str = param.String(doc="The fully qualified name of the model to train/test -e.g."
                                   "mymodule.configs.MyConfig.")
     azureml: bool = param.Boolean(False, doc="If True, submit the executing script to run on AzureML.")
+    tag: str = param.String(doc="A string that will be used as the display name of the run in AzureML.")
```

Comment:
The name of this param (and the corresponding key in the output of additional_run_tags) could be more informative - e.g. aml_display_name - since it isn't used for any other purpose?

Reply:
Ok let's keep it for consistency. Can make small improvements like this once we have everything in order